### ISSUE TITLE
Computing the ratio of the seasonal (three month) means

### ISSUE THREAD

**Issue by ivanhanigan** [swish-climate-impact-assessment/BiosmokeValidatedEvents#6]
@farhadsalimi, @ozjimbob The 2009 work for the 2011 paper used a rolling centred 90 day mean.
NB -45 and + 44 after reading the SAS CMOVAVE info as this is what it does when given an even number (90).  

Is this worth recreating in R?  Latest codes at https://github.com/swish-climate-impact-assessment/BiosmokeValidatedEvents/blame/master/inst/doc/impute_aphea2_sydney_pm25.Rmd#L200
use calendar seasons DJF (Summer), MAM, JJA, SON.

**Comment by farhadsalimi** [comment]
@ivanhanigan I don't believe it is really worth it as Katsouyanni et al, even used yearly ratio (http://pubs.healtheffects.org/getfile.php?u=518), page 10

**Comment by ivanhanigan** [comment]
I also believe it is not really worth it, but not because Katsouyanni used
yearly.  I suspect they did that because it was easier not because they
thought it wouldn't matter.
I think we should check what is difference between your 3 monthly calendar
season approach and my rolling 3-mo-centred approach.
But not this week.
